docs/pages/reference/alchemy/connectors-web/functions/alchemyAuth.mdx

```diff
@@ -19,31 +19,49 @@ import { alchemyAuth } from "@alchemy/connectors-web";
 
 ## Parameters
 
-### parameters
+### options
 
 `AlchemyAuthOptions`
 
 - Configuration for the connector
 
-### parameters.apiKey
+### options.apiKey
 
 `string`
 
 - API key for authentication with Alchemy services
 
-### parameters.iframeElementId
+### options.iframeElementId
 
 `string`
 
 - Optional ID for the iframe element used by Turnkey stamper
 
-### parameters.iframeContainerId
+### options.iframeContainerId
 
 `string`
 
 - Optional ID for the container element that holds the iframe
 
+### options.createTekStamper
+
+`CreateTekStamperFn`
+
+- Optional custom TEK stamper factory for React Native
+
+### options.createWebAuthnStamper
+
+`CreateWebAuthnStamperFn`
+
+- Optional custom WebAuthn stamper factory for passkey authentication
+
+### options.sessionConfig
+
+`object`
+
+- Session configuration bubbled straight to the signer
```

Comment:
nit: should be "auth session" instead of "signer". this might be resolved by regenerating docs